src/utils/tenderlyClient.ts

```diff
@@ -539,7 +539,7 @@ class Tenderly {
 }
 
 export const tenderly = new Tenderly(
-  process.env.TENDERLY_ACCESS_TOKEN,
-  process.env.TENDERLY_ACCOUNT,
-  process.env.TENDERLY_PROJECT_SLUG
+  process.env.TENDERLY_ACCESS_TOKEN!,
```

Comment:
weird, did you get a ts error before? As i think you should not: https://github.com/bgd-labs/aave-cli/blob/main/types.d.ts#L3

perhaps you do as your change on tsconfig only includes src? I'm not so familiar with all these config options.

Reply:
yup this was due to the tsconfig change, moving the types file into src also fixes